docs/src/main/sphinx/admin/fault-tolerant-execution.md

```diff
@@ -512,10 +512,10 @@ the property may be configured for:
   - HDFS
 :::
 
-It is recommended to set the `exchange.compression-codec` property to `LZ4` in
-the cluster's `config.properties` file, to reduce the exchange manager's overall
-I/O load. [](file-compression) is automatically performed and some details can
-be configured.
+To reduce the exchange manager's overall I/O load, the
+[](prop-exchange-compression-codec) configuration property is automatically
```

Comment:
a property can not be enabled .. it just is.. also does that link work?

Reply:
Okay, and yes the link works